.lighthouse/jenkins-x/release.yaml

```diff
@@ -21,6 +21,9 @@ spec:
           resources: {}
         - name: next-version
           resources: {}
+          script: |
+            #!/usr/bin/env bash
+            jx-release-version -semantic-release > VERSION
```

Comment:
this was needed to bump the version to v2 - I'll remove it after this PR is merged, because the new default will be to use semantic release